packages/nouns-api/src/services/ideas.ts

```diff
@@ -13,7 +13,15 @@ class IdeasService {
   static async get(id: number) {
     const idea = await prisma.idea.findUnique({
       where: { id: id },
-      include: { votes: true, comments: true },
+      include: {
+        votes: true,
+        comments: {
+          where: {
+            parentId: null,
+          },
```

Comment:
want to only fetch the comments with no parent, but include the replies.
This nests one level, unclear if it would let us nest another level as well? (If the reply comment has a reply, will it fetch that etc)

Easier for the UI to parse, because we fetch all the top level then recursively render the replies. Rather than fetching ALL comments and having to figure out their parent child ordering on our own